LDAP Authentication - Fail login attempts when wildcards are used (#38256)
demisto/content: Packs/OpenLDAP/ReleaseNotes/2_0_16.md

#### Integrations

##### LDAP Authentication

> - Fixed an issue where the ***ad-authenticate-and-roles*** command failed to disregard wildcard usage, resulting in unintended login successes.

Reviewer: ```suggestion
- Fixed an issue where the ***ad-authenticate-and-roles*** command did not filter wildcard symbols in usernames.
```
